test/benchmark/schbench/smp1/bench_results/p50_rps_smp1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "alert": {
+        "threshold": "130%",
+        "bigger_is_better": true
+    },
+    "result_extraction": {
+        "search_pattern": "\\*",
+        "nth_occurrence": 30,
+        "result_index": 3
+    },
+    "chart": {
+        "title": "[Scheduler] P50 RPS while max-loading the only CPU",
+        "description": "schbench -F 256 -n 5 -r 90",
+        "unit": "requests",
```

Comment:
I think it is `requests per second`.

Reply:
Thank you. It'll be fixed in next iteration.